enterprise/metrics/src/main/java/org/neo4j/metrics/source/ClusterMetrics.java

```diff
@@ -75,6 +96,30 @@ public Long getValue()
                     return monitor.lastAppliedTxId;
                 }
             } );
+
+        }
+    }
+
+    private boolean canResolveClusterMembersDependency()
```

Comment:
it would be really nice to have test that verify behaviour in case when there is no cluster members available